tests/integration/test_integration_snapshots.py

```diff
@@ -68,53 +88,88 @@ def process_trace(self, trace):
     tracer.shutdown()
 
 
+@pytest.mark.parametrize(
+    "writer",
+    ("default", "sync"),
+)
 @snapshot(include_tracer=True)
-def test_sampling(tracer):
+def test_sampling(writer, tracer):
+    if writer == "sync":
```

Comment:
nit we could parameterize the fixture: https://docs.pytest.org/en/stable/fixture.html#fixture-parametrize

Reply:
ah true